src/main/java/com/dev/cinema/controllers/MovieSessionController.java

```diff
@@ -0,0 +1,43 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.dto.MovieSessionRequestDto;
+import com.dev.cinema.model.dto.MovieSessionResponseDto;
+import com.dev.cinema.service.MovieSessionService;
+import com.dev.cinema.service.mapper.MovieSessionDtoMapper;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("movie-session")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionDtoMapper movieSessionDtoMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionDtoMapper movieSessionDtoMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionDtoMapper = movieSessionDtoMapper;
+    }
+
+    @PostMapping
+    public void addMovieSession(@RequestBody MovieSessionRequestDto dto) {
+        movieSessionService.add(movieSessionDtoMapper.fromRequestDto(dto));
+    }
+
+    @GetMapping("available")
```

Comment:
```suggestion
    @GetMapping("/available")
```
have you tested this?

Reply:
done